src/main/java/wolox/training/models/Book.java

```diff
@@ -11,7 +11,7 @@
 
   @Id
   @GeneratedValue(strategy = GenerationType.AUTO)
-  private long id;
+  private int id;
```

Comment:
Did you change to int for something in particular?

Reply:
I defined the repository with int instead of long

Reply:
Ok!